addons/mail/static/src/css/mail.css

```diff
@@ -1104,4 +1103,39 @@
     .openerp .oe_sidebar_suggestion .oe_suggest_title:hover .oe_suggestion_remove,
     .openerp .oe_sidebar_suggestion .oe_suggest_items:hover .oe_suggestion_remove_item {
         visibility: visible;
-    }
\ No newline at end of file
+    }
+/* ----------------------- @ Mention functionality ----------------------- */
+
+    .openerp .o_timeline div.mention {
+        position: absolute;
+        z-index: 499;
+        left: 5px;
+        margin: -7px 4px 0px 4px;
+        right: 30px;
+        box-shadow: 1px 2px 10px rgba(0, 0, 0, 0.648438);
+        -webkit-box-shadow: 1px 2px 10px rgba(0, 0, 0, 0.648438);
+        -moz-box-shadow: 1px 2px 10px rgba(0, 0, 0, 0.648438);
+    }
+    
+    .openerp .o_timeline div.mention > ul {
+        margin: 0;
+        padding: 0;
+    }
+    
+    .openerp .o_timeline div.mention > ul > li {
+        background-color: #FFF;
+        border: 1px solid #F0EEEE;
+        cursor: pointer;
+        list-style: none;
+        padding: 2px 0 2px 5px;
+    }
+    
+    .openerp .o_timeline .active {
+        background-color: #337AB7 !important;
```

Comment:
remove  !important, apply selector to only mention widget element not to timeline